node_build/builder.js

```diff
@@ -1053,6 +1053,7 @@ var configure = module.exports.configure = function (params, configFunc) {
             return;
         }
 
+if (process.env['NO_CODESTYLE'] != '1') {
```

Comment:
checking codestyle takes less than 1 second and it really doesn't check that much... Are you trying to improve performance or write garbage code ?